CONTRIBUTORS.txt

```diff
@@ -21,10 +22,16 @@ needs to be listed here.
 - Andrew Naylor <andrew_naylor@apple.com>
 - Andrius Aucinas <aaucinas@apple.com>
 - Ankit Aggarwal <ankit_aggarwal@apple.com>
+- Antoine Cœur <coeur@gmx.fr>
+- Antwan van Houdt <jabwd@exurion.com>
 - Arjun Nayini <aanayini@gmail.com>
 - Artem Redkin <aredkin@apple.com>
+- Austin <Austinpayne@users.noreply.github.com>
```

Comment:
@Austinpayne if you would like to have a real email address here, please add a comment with your preferred address. Thank you!

Reply:
austin.payne.01@gmail.com. Thank you!